### PR TITLE
Ajinkya cloudwatch encryption fix

### PR DESCRIPTION
**Pull request by ajinkya1986**
Changed the documentation for default encryption and created one for encryption with CMK

### CODE REVIEW

File: docs/aws/audit/cloudwatchmonitoring.mdx [hunk 1]
```diff
@@ -9,7 +9,8 @@ sidebar_label: "Cloudwatch Audit"
 - [Authorization Failures Alarm](cloudwatchmonitoring/rules/authorization_failures_alarm)
 - [AWS Config Changes Alarm](cloudwatchmonitoring/rules/aws_config_changes_alarm)
 - [CloudTrail Changes Alarm](cloudwatchmonitoring/rules/cloudtrail_changes_alarm)
-- [CloudWatch Log Groups Should Be Encrypted](cloudwatchmonitoring/rules/cloudwatch_log_groups_not_encrypted)
+- [CloudWatch Log Group Emcrypted](cloudwatchmonitoring/rules/cloudwatch_log_groups_not_encrypted)
```
Comment (mpurusottamc) [inline docs/aws/audit/cloudwatchmonitoring.mdx:12]: spelling mistake and title is wrong.

File: docs/aws/audit/cloudwatchmonitoring.mdx [hunk 2]
```diff
@@ -9,7 +9,8 @@ sidebar_label: "Cloudwatch Audit"
...
+- [CloudWatch Log Groups Should Be Encrypted With CMK](cloudwatchmonitoring/rules/cloudwatch_log_groups_with_cmk)
```
Comment (mpurusottamc) [inline docs/aws/audit/cloudwatchmonitoring.mdx:13]: did we not have this rule earlier?

Comment (ajinkya1986) [reply docs/aws/audit/cloudwatchmonitoring.mdx:13]: This policy document was not there, hence added it

**Review by mpurusottamc — APPROVED** [review]
lgtm

**Pull request merged**